pkg/minikube/cruntime/docker.go

```diff
@@ -688,6 +688,14 @@ const (
 	CNICacheDir = "/var/lib/cni/cache"
 )
 
+func getCriDockerdPath(cr CommandRunner) string {
+        rr, err := cr.RunCmd(exec.Command("which", "cri-dockerd"))
+        if err != nil {
+                return "/usr/bin/cri-dockerd"
```

Comment:
It's probably better to return an error fail early, where we can describe what happened, rather than in the systemd unit...

Reply:
I think we already looked for `cri-dockerd` in the user-facing command code, so this is not supposed to fail (at all)

Reply:
In that case, it seems better for everyone's collective sanity that an error is returned here, and that it does tear down the call stack/bubble up, as otherwise if the prelude to this code being invoked changes (and these assumptions are not obvious), severe bit-rot/surprising effects could ensue...

Reply:
Well, it was not _worse_ (than before)

Reply:
@afbjorklund how about returning the current default value in case of "which" failing, 
currently it assumes  "/usr/bin/cri-dockerd" in case "which" fails we dont even try the old way.

Reply:
That is the default.